### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Name property is empty for edit in "Push Image dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1408]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Push Image.
4. In "push Image" dialog ,tab to docker file edit and observe the name property using inspect.

**Actual Result:**
Name property is empty for the edit box

**Narrator behaviour:**
Tab
Editing

**NVDA behaviour:**
edit
blank

**Expected Result:**
Name property should be simple and meaningful.
Name property for the edit should be "Docker File".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![push image-edit](https://user-images.githubusercontent.com/22956130/35685605-3505d406-0790-11e8-9da7-73cc0da134a1.jpg)
![push image-edit-nvda](https://user-images.githubusercontent.com/22956130/35685611-37f9f57a-0790-11e8-8bc6-a66dc1bbf9ea.jpg)

**Comment by Eskibear** [comment]
fixed in PR #1590

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.

**Comment by WiproPSAccTest** [comment]
![1408-fixed](https://user-images.githubusercontent.com/22956130/39760838-102fe6b2-52f4-11e8-848f-cc931d1c7a5f.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749879
